src/commands/construct_transaction_command/sign_transaction/mod.rs

```diff
@@ -156,46 +155,75 @@ impl Submit {
+                                            println!("RPC Server Error: {}", debug_info);
+                                            return Ok(None)
+                                        }
+                                    }
+                                    near_jsonrpc_client::errors::JsonRpcServerError::RequestValidationError(rpc_request_validation_error) => {
+                                        println!("Incompatible request with the server: {:#?}",  rpc_request_validation_error);
+                                        return Ok(None)
+                                    }
+                                    near_jsonrpc_client::errors::JsonRpcServerError::InternalError{ info } => {
+                                        println!("Internal server error: {}.\nPlease wait. The next try to send this transaction is happening right now ...", info.unwrap_or_default());
+                                    }
+                                    near_jsonrpc_client::errors::JsonRpcServerError::NonContextualError(rpc_error) => {
+                                        println!("Unexpected response: {}", rpc_error);
+                                        return Ok(None)
+                                    }
+                                    near_jsonrpc_client::errors::JsonRpcServerError::ResponseStatusError(json_rpc_server_response_status_error) => match json_rpc_server_response_status_error {
+                                        near_jsonrpc_client::errors::JsonRpcServerResponseStatusError::Unauthorized => {
+                                            println!("JSON RPC server requires authentication. Please, authenticate near CLI with the JSON RPC server you use.");
+                                            return Ok(None)
+                                        }
+                                        near_jsonrpc_client::errors::JsonRpcServerResponseStatusError::TooManyRequests => {
+                                            println!("JSON RPC server is currently busy.\nPlease wait. The next try to send this transaction is happening right now ...");
+                                        }
+                                        near_jsonrpc_client::errors::JsonRpcServerResponseStatusError::Unexpected{status} => {
+                                            println!("JSON RPC server responded with an unexpected status code: {}", status);
+                                            return Ok(None);
+                                        }
                                     }
                                 }
-                                _ => println!("Unexpected response: {:#?}", err),
                             }
-                            return Ok(None);
+                            actix::clock::sleep(std::time::Duration::from_millis(100)).await;
```

Comment:
1. This code is duplicated in `src/commands/utils_command/send_signed_transaction/mod.rs`. We should avoid copy-pasted code. 
2. This error handling breaks the regular error handling flow and thus if something fails here near-cli-rs won't exist with non-zero exit code, and so it won't be a good citizen in scripts as it will just let the failing execution passed as "executed fine". We must return an error instead of `Ok(None)`.